web-client/src/utilities/delay.ts

```diff
@@ -0,0 +1,3 @@
+export const delay = async (timeToWait: number): Promise<void> => {
+  return await new Promise(resolve => setTimeout(resolve, timeToWait));
+};
```

Comment:
there already is a sleep, hrm, in fact two of these. do we really need a third?

* shared/src/business/utilities/sleep.ts
* shared/src/tools/helpers.ts .sleep